timeless/sms.py

```diff
@@ -13,6 +15,28 @@ def send(self):
         raise NotImplementedError
 
 
+class RetrySendSMS:
+    """Decorator class for resending requests"""
```

Comment:
@vladarefiev can you provide an example of how exactly this class will be used? Also, I would expect it to implement (or extend) abstract class ``SMS``, same as ``RedSMS``

Reply:
@amihaiemil this class is decorator. if you wish i can rewrite it to func based decorator